core-plugins/widgets/Cube-realtimesink.json

```diff
@@ -35,6 +46,17 @@
       "label": "CubeFact Mapping",
       "properties": [
         {
+          "widget-type": "keyvalue-dropdown",
+          "label": "Measurements",
+          "name": "cubeFact.measurements",
+          "description": "Fields that are used as measurements and the measurment type.",
```

Comment:
measurement field and aggregation field doesn't have to mach. aggregation group fields become context, while measurement is like metric name.  
Let's say there are two aggregation groups
[ {"agg1" : "remote_host", "agg2" : "response_code"}] and measurements [{"response_count" : "COUNTER"}, {"total_count" : "COUNTER"}]

if the input record has values for these fields <remote_host, response_code, response_count> we would emit 2 measures, one for each aggregation group.

if input record has only these field values <remote_host, response_code> we wouldn't any measures, as none of the measurement fields are present in this record.

i can add this in reference doc.